pipeline/ticker.go

```diff
@@ -116,21 +135,6 @@ func checkActivePipelines() {
 				continue
 			}
 
-			// We couldn't find the pipeline. Create a new one.
```

Comment:
What happens when someone locally builds a pipeline and puts it into the pipelines folder?
I know, many people will use the "create pipeline" functionality but I would still like to support that use-case. 🤗

Reply:
Uh! That's something I didn't think about. :D Thanks! I'll add that.

Reply:
Addressed. Now I understand Gaia a bit better. :) Thanks for that.